common/io/src/kseqpp_fasta_parser.hpp

```diff
@@ -22,20 +22,28 @@ namespace io
 class FastaParserKseqpp : public FastaParser
 {
 public:
-    FastaParserKseqpp(const std::string& fasta_file, int min_sequence_length, bool shuffle);
-
-    int32_t get_num_seqences() const override;
-
-    FastaSequence get_sequence_by_id(int32_t i) const override;
-
-    std::vector<std::pair<int, int>> get_read_chunks(int max_chunk_size) const override;
+    /// \brief Constructor
+    /// \param fasta_file Path to FASTA(.gz) file. If .gz, it must be zipped with bgzip.
+    /// \param min_sequence_length Minimum length a sequence needs to be to be parsed. Shorter sequences are ignored.
+    /// \param shuffle Enables shuffling reads
+    FastaParserKseqpp(const std::string& fasta_file,
+                      const number_of_basepairs_t min_sequence_length,
```

Comment:
`const` call-by-value in declaration (see above).

Reply:
done